dashboard/app/views/plc/learning_modules/new_learning_resource_for_module.html.haml

```diff
@@ -0,0 +1,37 @@
+- content_for(:head) do
+  = stylesheet_link_tag 'css/levelbuilder', media: 'all'
+
+%h3{style: 'margin-bottom: 24px;'}
+  Add a resource (Module: #{@learning_module.name})
+
+.new_resource_section{style: 'float: left; width: 33%'}
+  %p
+    = link_to 'New Script', new_script_path, class: 'btn'
+  %p
+    Select Script if
+    %li
+      You want to create multiple screens that a learner will click through
+    %li
+      You will incorporate multiple puzzles or questions
+
+.new_resource_section{style: 'float: right; width: 33%'}
+  %p
+    = link_to 'New Link', new_plc_task_path(plc_learning_module_id: @learning_module.id, type: 'Plc::LearningResourceTask'), class: 'btn'
```

Comment:
Learning resources are types of tasks.

You can think of tasks as things that a user should do in order to complete a module. There are script completion tasks (do this script), written assignment tasks (write a paragraph), and learning resources which are links to helpful external content - videos, blog posts, etc.

So someone who wants to create a new link to an external resource should create a learning resource task - going to plc_task_path with the above params will pre-fill them for the user.